AlphaPass.sol

```diff
@@ -7,25 +7,63 @@ import "erc721a/contracts/ERC721A.sol";
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 
 contract AlphaNft is Ownable, ERC721A, ReentrancyGuard {
-    uint256 public immutable maxQty = 2000;
-    uint256 public immutable firstFree = 1000;
+    uint256 public immutable maxQty = 1024;
```

Comment:
can make it all modifiable. No need to be immutable unless you're not gonna change them.